tests/Test/Disciplina/DB/DSL/Interpret/SimpleTxDB.hs

```diff
@@ -0,0 +1,190 @@
+module Test.Disciplina.DB.DSL.Interpret.SimpleTxDB where
+
+import Test.Common
+
+import Crypto.Error (CryptoFailable(..))
+import Disciplina.DB (QueryTx(..), QueryTxs(..), ObjHashEq(..), WHERE(..)
+                     ,TxIdEq(..), TxGrade(..), QueryObj(..), Obj
+                     ,TxsFilterExpr(..), runSimpleTxDBQuery)
+import Disciplina.Educator (PrivateTx(..), EducatorTxMsg(..)
+                           ,StudentTxMsg(..), PrivateTxPayload(..))
+import Disciplina.Crypto (hash, AbstractPK(..), AbstractSK(..), PublicKey, SecretKey)
+import Disciplina.Core (CourseId(..), SubjectId, Grade(..), mkAddr)
+
+import qualified Crypto.PubKey.Ed25519 as Ed25519
+import qualified Data.ByteString.Char8 as C
+
+type StudentKey = Char
+type EducatorKey = Char
+
+-- | SubjectIds are taken from Disciplina.Core.ATG
+sIdMathematics, sIdComputerScience, sIdElementary
+   ,sIdCalculi, sIdLogic, sIdEngineering
+   ,sIdTheory, sIdHighSchoolAlgebra, sIdPiCalculus
+   ,sIdComputabilityTheory :: SubjectId
+
+sIdMathematics = 1
+sIdComputerScience = 2
+sIdElementary = 3
+sIdCalculi = 4
+sIdLogic = 5
+sIdEngineering = 6
+sIdTheory = 7
+sIdHighSchoolAlgebra = 8
+sIdPiCalculus = 9
+sIdComputabilityTheory = 10
+
+-- | Create a private transaction
+mkPrivateTx :: CourseId -> PrivateTxPayload -> StudentKey -> EducatorKey -> PrivateTx
+mkPrivateTx courseId payload studentKey educatorKey =
+  PrivateTx {
+       _ptxStudentId = mkAddr . fst $ mkKeyPair studentKey
+     , _ptxCourseId = courseId
+     , _ptxEducatorId = mkAddr . fst $ mkKeyPair educatorKey
+     , _ptxPayload = payload
+     }
+ where
+   -- Create key pair from seed
+   mkKeyPair :: Char -> (PublicKey, SecretKey)
```

Comment:
Thinking about it, I understood that there should possibly be a method in `SignatureScheme` class for keypair generation from a seed. But this is out of scope of this PR.